trin-core/src/jsonrpc/handlers.rs

```diff
@@ -22,44 +24,56 @@ pub struct JsonRpcHandler {
 
 impl JsonRpcHandler {
     pub async fn process_jsonrpc_requests(mut self) {
-        while let Some(request) = self.portal_jsonrpc_rx.recv().await {
-            let response: Value = match request.endpoint {
+        while let Some(portal_jsonrpc_req) = self.portal_jsonrpc_rx.recv().await {
+            let response: Value = match portal_jsonrpc_req.endpoint {
                 TrinEndpoint::Discv5Endpoint(endpoint) => match endpoint {
                     Discv5Endpoint::NodeInfo => self.discovery.read().await.node_info(),
                     Discv5Endpoint::RoutingTableInfo => {
                         self.discovery.write().await.routing_table_info()
                     }
+                    Discv5Endpoint::FindNodes => {
+                        // TODO: Add RPC handler for every endpoint and validate parameters
```

Comment:
I do agree, parameter validation for all incoming requests is important and needs to be done. I'm a bit confused what you mean by add RPC handler for every endpoint? As I understand, in this repo the term "handler" is meant for intermediary objects that receive, match and fulfill rpc requests to their domain `portalState_*`, `portalHistory_*` etc.

Reply:
By handlers for every endpoint, I mean a data structures responsible for doing the endpoint validations similar to those in [ddht](https://github.com/ethereum/ddht/blob/master/ddht/v5_1/rpc_handlers.py).